src/spectrecoff/Prompt.fs

```diff
@@ -40,7 +46,10 @@ let chooseFrom choices question =
     prompt (Prompts.selectionPrompt question choices)
 
 let chooseMultipleFrom choices question = 
-    prompt (Prompts.multiSelectionPrompt question choices)
+    prompt (Prompts.multiSelectionPrompt question choices defaultMultiSelectionOptions)
```

Comment:
If you change the order, you can define the default one as a partial application:

```
let chooseMultipleFrom choices question =
    chooseMultipleFromWith defaultMultiSelectionOptions
```

Reply:
That's a good simplification - will do